iphone/Classes/TiContactsGroup.m

```diff
@@ -211,7 +214,10 @@ -(NSArray*)sortedMembers:(id)value
 		fetchRequest.sortOrder = sortOrder;
 		fetchRequest.mutableObjects = YES;
 		BOOL success = [ourContactStore enumerateContactsWithFetchRequest:fetchRequest error:&error usingBlock:^(CNContact * __nonnull contact, BOOL * __nonnull stop) {
-			TiContactsPerson* person = [[[TiContactsPerson alloc] _initWithPageContext:[self executionContext] contactId:(CNMutableContact*)contact module:module] autorelease];
+			TiContactsPerson* person = [[[TiContactsPerson alloc] _initWithPageContext:[self executionContext]
+                                                                             contactId:(CNMutableContact*)contact
+                                                                                module:module
+                                                                              observer:module] autorelease];
```

Comment:
@hansemannn observer is `module` because we want to propagate all changes to the `ContactsModule` where saveRequest is updated.